LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-relative-color.html

```diff
@@ -755,6 +755,10 @@
     fuzzy_test_valid_color(`lch(from var(--mycolor) l 0 h)`);
     fuzzy_test_valid_color(`var(--mygray)`);
     fuzzy_test_valid_color(`lch(from var(--mygray) l 30 h)`);
+
+    // Ensure that converting between legacy and modern sRGB color spaces work as expected.
+    fuzzy_test_valid_color(`color(from rebeccapurple srgb r g b)`, `color(srgb 0.4 0.2 0.6)`, 0.01);
```

Comment:
I think this is incorrect given the accepted terms of https://github.com/w3c/csswg-drafts/issues/10328.

Reply:
I wanted to look into this line and the next one, thanks for chiming in!
With your implementation, they do fail. Also, I think they're adequately covered by tests above, so I'll remove them.